playbooks/roles/dynflow_devel/tasks/git.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: 'Install git'
+  become: true
+  yum: name=git-core state=present
```

Comment:
How do I reuse tasks? i.e. this is directly copied from [hammer_devel](https://github.com/theforeman/forklift/blob/master/playbooks/roles/hammer_devel/tasks/git.yml)